From 0f073c51ae4d4b7037222620c716f54e181d3ae2 Mon Sep 17 00:00:00 2001
From: firnsy <firnsy@securixlive.com>
Date: Sat, 18 Aug 2012 16:00:32 +1000
Subject: [PATCH 1/2] cleanup: whitespace fixes for progress screen.

---
 pyanaconda/iw/progress_gui.py | 26 ++++++++++++--------------
 1 file changed, 12 insertions(+), 14 deletions(-)

diff --git a/pyanaconda/iw/progress_gui.py b/pyanaconda/iw/progress_gui.py
index 93d4970..516cc30 100644
--- a/pyanaconda/iw/progress_gui.py
+++ b/pyanaconda/iw/progress_gui.py
@@ -37,7 +37,7 @@ class InstallProgressWindow (InstallWindow):
     windowTitle = N_("Installing Packages")
 
     def __init__ (self, ics):
-	InstallWindow.__init__ (self, ics)
+        InstallWindow.__init__ (self, ics)
         ics.setPrevEnabled (False)
         ics.setNextEnabled (False)
 
@@ -106,17 +106,16 @@ class InstallProgressWindow (InstallWindow):
             files = ["progress_first.png"]
 
         return files
-        
 
     def getScreen (self, anaconda):
-	self.intf = anaconda.intf
-	if anaconda.dir == DISPATCH_BACK:
-	    self.intf.icw.prevClicked()
-	    return
+        self.intf = anaconda.intf
+        if anaconda.dir == DISPATCH_BACK:
+            self.intf.icw.prevClicked()
+            return
 
         self.pixmaps = self._getRnotes()
 
-	# Create vbox to contain components of UI
+        # Create vbox to contain components of UI
         vbox = gtk.VBox (False, 12)
 
         # Create rnote area
@@ -133,18 +132,17 @@ class InstallProgressWindow (InstallWindow):
             frame.add(box)
             vbox.pack_start(frame, False)
 
-
-	self.progress = gtk.ProgressBar()
+        self.progress = gtk.ProgressBar()
         vbox.pack_start(self.progress, False)
 
         self.infolabel = gui.WrappingLabel("")
         self.infolabel.set_alignment(0,0)
         vbox.pack_start(self.infolabel)
 
-	# All done with creating components of UI
-	self.intf.setPackageProgressWindow(self)
-	self.intf.setInstallProgressClass(self)
+        # All done with creating components of UI
+        self.setPackageProgressWindow(self)
+        self.intf.setInstallProgressClass(self)
 
-	vbox.set_border_width(6)
+        vbox.set_border_width(6)
 
-	return vbox
+        return vbox
-- 
1.7.11.4

