diff -Nurd anaconda-17.28-orig/pyanaconda/bootloader.py anaconda-17.28/pyanaconda/bootloader.py
--- anaconda-17.28-orig/pyanaconda/bootloader.py	2012-05-17 06:27:58.000000000 +1000
+++ anaconda-17.28/pyanaconda/bootloader.py	2012-05-25 12:01:39.239455247 +1000
@@ -1649,7 +1649,7 @@
 
             if not ret:
                 self.errors.append(_("Your BIOS-based system needs a special "
-                                     "partition to boot with Fedora's new "
+                                     "partition to boot with Kororaa's new "
                                      "disk label format (GPT). To continue, "
                                      "please create a 1MB 'BIOS Boot' type "
                                      "partition."))
diff -Nurd anaconda-17.28-orig/pyanaconda/installclasses/kororaa.py anaconda-17.28/pyanaconda/installclasses/kororaa.py
--- anaconda-17.28-orig/pyanaconda/installclasses/kororaa.py	1970-01-01 10:00:00.000000000 +1000
+++ anaconda-17.28/pyanaconda/installclasses/kororaa.py	2012-05-25 11:58:39.658909466 +1000
@@ -0,0 +1,146 @@
+#
+# kororaa.py
+#
+# Copyright (C) 2007  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from pyanaconda.installclass import BaseInstallClass
+from pyanaconda.constants import *
+from pyanaconda.product import *
+from pyanaconda.flags import flags
+from pyanaconda import iutil
+from pyanaconda.network import hasActiveNetDev
+from pyanaconda import isys
+
+import os, types
+import gettext
+_ = lambda x: gettext.ldgettext("anaconda", x)
+
+from pyanaconda import installmethod
+from pyanaconda import yuminstall
+
+import rpmUtils.arch
+from decimal import Decimal
+
+class InstallClass(BaseInstallClass):
+    # name has underscore used for mnemonics, strip if you dont need it
+    id = "kororaa"
+    name = N_("_Kororaa")
+    _description = N_("The default installation of %s includes a set of "
+                      "software applicable for general internet usage. "
+                      "You can optionally select a different set of software "
+                      "now.")
+    _descriptionFields = (productName,)
+    sortPriority = 10000
+    if productName.startswith("Red Hat Enterprise"):
+        hidden = 1
+
+    tasks = [(N_("Graphical Desktop"),
+              ["admin-tools", "base", "base-x", "core", "editors", "fonts",
+               "games", "gnome-desktop", "graphical-internet", "graphics",
+               "hardware-support", "input-methods", "java", "office",
+               "printing", "sound-and-video", "text-internet"]),
+             (N_("Software Development"),
+              ["base", "base-x", "core", "development-libs",
+               "development-tools", "editors", "fonts", "gnome-desktop",
+               "gnome-software-development", "graphical-internet", "graphics",
+               "hardware-support", "input-methods", "java", "sound-and-video", "text-internet",
+               "x-software-development"]),
+             (N_("Web Server"),
+              ["admin-tools", "base", "base-x", "core", "editors",
+               "gnome-desktop", "graphical-internet", "hardware-support",
+               "java", "text-internet", "web-server"]),
+             (N_("Minimal"), ["core"])]
+
+    _l10n_domain = "anaconda"
+
+    def getPackagePaths(self, uri):
+        if not type(uri) == types.ListType:
+            uri = [uri,]
+
+        return {'Installation Repo': uri}
+
+    def configure(self, anaconda):
+	BaseInstallClass.configure(self, anaconda)
+        BaseInstallClass.setDefaultPartitioning(self,
+                                                anaconda.storage,
+                                                anaconda.platform)
+
+    def setGroupSelection(self, anaconda):
+        BaseInstallClass.setGroupSelection(self, anaconda)
+        map(lambda x: anaconda.backend.selectGroup(x), ["core"])
+
+    def getBackend(self):
+        if flags.livecdInstall:
+            import pyanaconda.livecd
+            return pyanaconda.livecd.LiveCDCopyBackend
+        else:
+            return yuminstall.YumBackend
+
+    def productMatches(self, oldprod):
+        if oldprod is None:
+            return False
+
+        if oldprod.startswith(productName):
+            return True
+
+        productUpgrades = {
+                "Kororaa": ("Fedora", ),
+                "Fedora Core": ("Red Hat Linux", ),
+                "Fedora": ("Fedora Core", )
+        }
+
+        if productUpgrades.has_key(productName):
+            acceptable = productUpgrades[productName]
+        else:
+            acceptable = ()
+
+        for p in acceptable:
+            if oldprod.startswith(p):
+                return True
+
+        return False
+
+    def versionMatches(self, oldver):
+        if oldver is None:
+            return False
+
+        try:
+            oldVer = Decimal(oldver)
+            # Trim off any "-Alpha" or "-Beta".
+            newVer = Decimal(productVersion.split('-')[0])
+        except Exception:
+            return True
+
+        # This line means we do not support upgrading from anything older
+        # than two versions ago!
+        return newVer >= oldVer and newVer - oldVer <= 2
+
+    def setNetworkOnbootDefault(self, network):
+        # if something's already enabled, we can just leave the config alone
+        for devName, dev in network.netdevices.items():
+            if dev.get('ONBOOT') == 'yes':
+                return
+
+        # the default otherwise: bring up the first wired netdev with link
+        for devName, dev in network.netdevices.items():
+            if (not isys.isWirelessDevice(devName) and
+                isys.getLinkStatus(devName)):
+                dev.set(('ONBOOT', 'yes'))
+                break
+
+    def __init__(self):
+	BaseInstallClass.__init__(self)
diff -Nurd anaconda-17.28-orig/pyanaconda/packages.py anaconda-17.28/pyanaconda/packages.py
--- anaconda-17.28-orig/pyanaconda/packages.py	2012-05-16 08:01:27.000000000 +1000
+++ anaconda-17.28/pyanaconda/packages.py	2012-05-25 12:06:07.038351706 +1000
@@ -269,7 +269,7 @@
         v = get_version(h)
         tag = get_tag(h)
         if v == "" or tag == "":
-            log.warning("Unable to determine kernel type/version for %s-%s-%s.%s" %(h['name'], h['version'], h['release'], h['arch'])) 
+            log.warning("Unable to determine kernel type/version for %s-%s-%s.%s" %(h['name'], h['version'], h['release'], h['arch']))
             continue
         # rpm really shouldn't return the same kernel more than once... but
         # sometimes it does (#467822)
@@ -308,7 +308,8 @@
     publicBetas = { "Red Hat Linux": "Red Hat Linux Public Beta",
                     "Red Hat Enterprise Linux": "Red Hat Enterprise Linux Public Beta",
                     "Fedora Core": "Fedora Core",
-                    "Fedora": "Fedora" }
+                    "Fedora": "Fedora",
+                    "Kororaa": "Kororaa" }
 
 
     if anaconda.dir == DISPATCH_BACK:
@@ -320,7 +321,7 @@
             fileagainst = val
     if fileagainst is None:
         fileagainst = "%s Beta" %(productName,)
-    
+
     while 1:
 	rc = anaconda.intf.messageWindow(_("Warning"),
 				 _("Warning!  This is pre-release software!\n\n"
